addons/report/models/ir_actions_report_xml.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class IrActionsReport(models.Model):
+    _inherit = 'ir.actions.report.xml'
+
+    paperformat_id = fields.Many2one('report.paperformat', 'Paper format')
+
+    @api.multi
+    def associated_view(self):
+        """Used in the ir.actions.report.xml form view in order to search naively after the view(s)
+        used in the rendering.
+        """
+        try:
+            view_action = self.env.ref('base.action_ui_view', raise_if_not_found=True)
```

Comment:
you already passed "raise_if_not_found=True" which raise validation when no id found. no more try and except required. Remove it.